aiida/cmdline/commands/cmd_config.py

```diff
@@ -190,3 +194,31 @@ def verdi_config_caching(disabled):
                     echo.echo(identifier)
             elif disabled:
                 echo.echo(identifier)
+
+
+@verdi_config.command('downgrade')
+@click.argument('version', type=int)
+@click.option(
```

Comment:
Correct me if I am wrong, but I don't think you answered my question yet why we need the option to specify an arbitrary input and output file. Since this is user facing it should simply always operate on the currently configured config file. I would suggest to remove this additional complexity since I don't think it is needed for now and you don't have tests for it.

Reply:
removed